manifold_airflow_dags/tupress_ingest_dag.py

```diff
@@ -84,6 +87,19 @@ def calculate_file_to_move(**context):
     dag=DAG
 )
 
+ingest_deltas_bash = f"""
+sudo su - { TUPRESS_USER_NAME } bash -c \
+ "cd /var/www/tupress &&\
+ RAILS_ENV=production bundle exec rake db:seed:run_updates['%s']"
+""" % f"{ TUPRESS_WEB_PATH }/%s" % "{{ ti.xcom_pull(task_ids='get_file_to_transfer') }}"
```

Comment:
@nomadicoder, I believe there is still an issue with this, now on line 94:

example
```
f"test {foo} %s" % "hello %s" & "world"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for &: 'str' and 'str'
```
I don't think. you can use the formatter `&` operator in double like that.

Could you please add an example of exactly what you want this to output?  It might help to create a test case here.

Without knowing exactly what you are trying to do I could suggest the following

```suggestion
ingest_deltas_bash = f"""
sudo su - { TUPRESS_USER_NAME } bash -c \
 "cd /var/www/tupress &&\
 RAILS_ENV=production bundle exec rake db:seed:run_updates['%s']"
""" % f"{ TUPRESS_WEB_PATH }/" + "{{ ti.xcom_pull(task_ids='get_file_to_transfer') }}"
```

Reply:
Actually, I'm wrong.. I replace % with &